modules/nodes-descheduler-profiles.adoc

```diff
@@ -61,17 +61,22 @@ Do not enable both `SoftTopologyAndDuplicates` and `TopologyAndDuplicates`. Enab
 `EvictPodsWithLocalStorage`:: This profile allows pods with local storage to be eligible for eviction.
 
 `EvictPodsWithPVC`:: This profile allows pods with persistent volume claims to be eligible for eviction. If you are using `Kubernetes NFS Subdir External Provisioner`, you must add an excluded namespace for the namespace where the provisioner is installed.
+
+`CompactAndScale`:: This profile enables the `HighNodeUtilization` strategy, which attempts to evict pods from underutilized nodes to allow a workload to run on a smaller set of nodes. A node is considered underutilized if its usage is below 20% for all thresholds (CPU, memory, and number of pods).
+
 endif::nodes[]
 ifdef::virt[]
 Use the Technology Preview `DevPreviewLongLifecycle` profile to enable the descheduler on a virtual machine. This is the only descheduler profile currently available for {VirtProductName}. To ensure proper scheduling, create VMs with CPU and memory requests for the expected load.
+endif::virt[]
```

Comment:
`DevPreviewLongLifecycle` is deprecated in favor of `LongLifecycle`.

Reply:
@ingvagabund ack - I could change this, but I left it alone because this text only appears in the CNV docs. So I am leaving it for the CNV writers to update. I'm still waiting to hear back on someone to coordinate with, but we'll make sure it gets updated somehow.

I'm also not announcing the deprecation of DevPreviewLongLifecycle in the descheduler release notes, since it was never mentioned in our docs at all. CNV will take care of announcing that deprecation in their RNs

Reply:
Gotcha :)